### ISSUE TITLE
DOC: Example missing in pandas.DataFrame.to_html

### ISSUE THREAD

**Issue by vimaloctavius** [pandas-dev/pandas#44945]
### Pandas version checks

- [X] I have checked that the issue still exists on the latest versions of the docs on `master` [here](https://pandas.pydata.org/docs/dev/)

### Location of the documentation

pandas.DataFrame.to_html (https://pandas.pydata.org/docs/dev/reference/api/pandas.DataFrame.to_html.html?highlight=pandas%20dataframe%20to_html#)

### Documentation problem

Came across the documentation for pandas.DataFrame.to_html, found no example

### Suggested fix for documentation

Example:
>>> df = pd.DataFrame({'ID': ['XYZ', 'ABC'],
...                     'Department': ['HR', 'Admin'],
...                     'Job Title': ['Title1', 'Title2']})

>>> print(df.to_html())
<table border="1" class="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th></th>
      <th>ID</th>
      <th>Department</th>
      <th>Job Title</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>0</th>
      <td>XYZ</td>
      <td>HR</td>
      <td>Title1</td>
    </tr>
    <tr>
      <th>1</th>
      <td>ABC</td>
      <td>Admin</td>
      <td>Title2</td>
    </tr>
  </tbody>
</table>

**Comment by phofl** [comment]
Would you be interested in submitting a pr?

**Comment by vimaloctavius** [comment]
Yes, it'll be by first one, I've read the guidelines will shout out if I need any help 😊. Thanks

**Comment by vimaloctavius** [comment]
> Would you be interested in submitting a pr?

Yes, it'll be by first one, I've read the guidelines will shout out if I need any help 😊. Thanks

**Comment by vimaloctavius** [comment]
take

**Comment by vimaloctavius** [comment]
@phofl I've updated the example in my locally cloned repo (pandas/pandas/core/frame.py). How do I test it so I'm sure before submitting the PR. As I'm new to coding space, any help is appreciated. Thanks much!

**Comment by phofl** [comment]
You can check the contributing Guide. There is a section about building the docs locally

**Comment by vimaloctavius** [comment]
I was able to create the pandas-dev environment. I ran:   "python scripts/validate_docstrings.py pandas.DataFrame.to_html" Received the below validation output: 
3 Errors found:
	No extended summary found
	Unknown parameters {'min_rows'}
	No examples section found
----------------
Please see screenshot attached. In pandas/core/frame.py, I've added the example from Line 2907 onwards.
What am I missing? Thanks!
![Screenshot from 2021-12-28 01-00-29](https://user-images.githubusercontent.com/13742673/147549714-a797ac1b-22f8-4d98-ac69-582aeb9bc24e.png)

**Comment by ParfaitG** [comment]
Carefully read the [docstring guide](https://pandas.pydata.org/docs/development/contributing_docstring.html#section-7-examples), particularly on _Examples_. Several issues with your attempt:

- Example**s** section should be plural, have no colon ending, and a line break before the section. 
- Remove `print()` for output purposes. 
- Curly `{...}` braces in `DataFrame()` constructor must be doubled to escape it as formatting symbol.
- Previous item unrelated to your addition needs adjustment: remove `min_rows` lines (deprecated arg).

See other docstrings for guidance.

**Comment by vimaloctavius** [comment]
Thank you so much, working on it!

**Comment by vvaidy** [comment]
I'll take this one and push it later tonight assuming i dont hit any problems @noatamir and @phofl

**Comment by vvaidy** [comment]
@phofl ... Pushed the recommended fix (above)

**Comment by ArinaKostanyan** [comment]
take

**Comment by ArinaKostanyan** [comment]
Is this still active like there is a need to make changes or the html string directly being outputed is what we need ?

**Comment by itz-goutham** [comment]
Is this issue still open?
If yes, I am willing to take it.

**Comment by itz-goutham** [comment]
take

**Comment by itz-goutham** [comment]
@jbrockmendel Can you please assign me this issue?
